### ISSUE TITLE
poetry self:update does not update to the latest version

### ISSUE THREAD

**Issue by Tonkonozhenko** [sdispater/poetry#171]
Hi,

Just tried to update
```
➜  ~ poetry self:update
Updating to 0.10.1
  - Getting dependencies
  - Vendorizing dependencies
  - Updating poetry

poetry (0.10.1) successfully installed!
```

**Comment by sdispater** [comment]
Yes, I noticed this yesterday and fixed it in commit 043b9ec

For the time being you can update to `0.10.2` by executing:

```
poetry self:update 0.10.2
```

**Comment by sdispater** [comment]
This should be fixed in the latest release (`0.10.3`)
